packages/clay-link/src/ClayLink.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+
+type DisplayType = 'secondary';
+
+type ButtonDisplayType = 'link' | 'primary' | 'secondary';
+
+interface Props extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+	buttonDisplayType?: ButtonDisplayType;
```

Comment:
Hmm not sure about this. Seems weird to allow people to style a link like a button. I noticed on the clayui site we don't have this listed and on the lexicon site it even specifies that we need to make distinctions between links and buttons. I'm guessing you just copied this over from v2 Link?

Reply:
For cases like that since it is more of a one-off case, what do you think about just isolating that functionality to that component itself? So for that button, rather than use `ClayLink`, you would just manually add an anchor tag with the button classes. That way it is isolated to its use case and then we can keep our generic components more in sync with lexicon/clay docs.

Reply:
Yes I think it is better, I will remove this, I do not remember if we had other cases for this, but if it does appear we can think of other solutions.